### PR TITLE
Nero qt6 buildroot

### PR DESCRIPTION
**Pull request by jr1221**
Draft PR.
This adds qt6 support to buildroot, and builds nero with it.  PR based on `buildroot-ap` branch as it adds various improvements I needed for this.  Nero is a submodule to allow easy version pinning and switching branches.  

`nerqt6`:
In order to use external tree, adds a nerqt6 as an alternative to qt6 with the NER changes merged in.  It replicates source qt6 package as closely as possible.  Major uncommented change is bumping the version from 6.4.3 to 6.5.3, and adding a patch (from upstream) to fix a bug introduced in 6.5.  Also refactoring qt6 to nerqt6 is uncommented as that is abundant.  Comments about functional changes from me start with `NER:` and explain where the source file is and what was changed.  The eventual goal is removal of nerqt6 (basically revert ae6184a0eba07c2f76dac9893f079a5f0dd51e72) as upstream picks up qt6shadertools (current patch in progress rn) and eventual adds qt6declarative.

`nero2`:
First, using the `auto-modloader` package introduced in the original `nrc-build-v2`, 4 modules are probed to enable the pi 4 gpu and hdmi outputs.  Then the startup script runs, which boots NEROApp in eglfs mode with the `/etc/eglfs.json` config file specified.  More on that file and other config options [here](https://doc.qt.io/qt-6/embedded-linux.html).  This file should be edited to customize resolution, refresh rate, and input sources in the future.  Current functionality overrides the console and outputs hdmi on the same screen in 3d rendering mode.  Therefore on boot the console log and a login prompt is shown until `S99nero2` runs and shows the dashboard.  Since keyboard input is disabled the app cannot be restarted without console access from ssh.

To apply nero changes without clean make:
- Edit nero2 as needed in the nero-2.0 repository
- update the submodule to that commit/branch in the Siren git repository.
- Run `make nero2-reconfigure` in the builroot submodule to completely resource and rebuild the package
- Replace `/usr/bin/NEROApp` on the TPU with the newly built one at `buildroot/output/target/usr/bin/NEROApp` using scp or other tool.
- Run `/etc/init.d/S99nero2 restart` or reboot the system.

### CODE REVIEW

File: odysseus/odysseus_tree/configs/raspberrypi4_64_tpu_defconfig [hunk 1]
```diff
@@ -21,7 +22,16 @@ BR2_LINUX_KERNEL_INTREE_DTS_NAME="broadcom/bcm2711-rpi-4-b"
 BR2_LINUX_KERNEL_NEEDS_HOST_OPENSSL=y
 BR2_PACKAGE_LINUX_TOOLS_GPIO=y
 BR2_PACKAGE_BUSYBOX_SHOW_OTHERS=y
-BR2_PACKAGE_XZ=y
+BR2_PACKAG
```
Comment (nwdepatie) [inline odysseus/odysseus_tree/configs/raspberrypi4_64_tpu_defconfig:25]: Is this a typo?

Comment (jr1221) [reply odysseus/odysseus_tree/configs/raspberrypi4_64_tpu_defconfig:25]: I have no idea how but it seems to be a result of the merge.  That package not being there would be very bad.

Comment (nwdepatie) [reply odysseus/odysseus_tree/configs/raspberrypi4_64_tpu_defconfig:25]: Might've been my bad :grimacing:

Comment (jr1221) [reply odysseus/odysseus_tree/configs/raspberrypi4_64_tpu_defconfig:25]: Also this file should be deleted: odysseus/odysseus_tree/overlays/rootfs_overlay_common/etc/modules

Ill commit a fix.

Comment (nwdepatie) [reply odysseus/odysseus_tree/configs/raspberrypi4_64_tpu_defconfig:25]: Any chance you can add back in?

Comment (nwdepatie) [reply odysseus/odysseus_tree/configs/raspberrypi4_64_tpu_defconfig:25]: Thank you!

Comment (jr1221) [reply odysseus/odysseus_tree/configs/raspberrypi4_64_tpu_defconfig]: All right should be all set.  Ig github being weird.

**Review by nwdepatie — APPROVED** [review]
LGTM and built locally for me

**Pull request merged**